src/main/java/org/myrobotlab/service/interfaces/IKJointAnglePublisher.java

```diff
@@ -1,12 +1,11 @@
 package org.myrobotlab.service.interfaces;
 
-import java.util.HashMap;
-import java.util.Map;
+import org.myrobotlab.service.data.AngleData;
 
 public interface IKJointAnglePublisher {
 
   public String getName();
 
-  public Map<String, Double> publishJointAngles(HashMap<String, Double> angleMap);
+  public AngleData publishJointAngles(AngleData angle);

```

Comment:
not plural in this refactor.  Also,  I though we were creating a new interface.. not modifying this existing one..

Reply:
fixed to singular, no there was no mention of creating a new interface or its the name of that interface